optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/roster/RosterRestService.java

```diff
@@ -30,6 +30,14 @@
 @GenRestBuilder
 public interface RosterRestService {
 
+    // ************************************************************************
+    // RosterState
+    // ************************************************************************
+    @ApiOperation("Fetches the current Roster State")
```

Comment:
All the other ApiOperations use the wording "get".
Even in the swagger docs, it's nice to use consistent wording, so when a different word is used, we signal we mean there's some sort of difference with the other methods.